tools/wake-format/emitter.cpp

```diff
@@ -1361,9 +1399,14 @@ wcl::doc Emitter::walk_binary(ctx_t ctx, CSTElement node) {
 
   std::vector<CSTElement> parts = {};
   if (is_op_left_assoc(op_token)) {
-    parts = collect_left_binary(op_token, node);
+    if (is_binop_matching_str(op_token, TOKEN_OP_OR, "|>") ||
+        is_binop_matching_str(op_token, TOKEN_OP_OR, "|<")) {
+      parts = collect_left_binary({{TOKEN_OP_OR, "|>"}, {TOKEN_OP_OR, "|<"}}, node);
```

Comment:
This is the sort of thing we'll need to fix `,` and `++`, right?

Reply:
I think so, not 100% sure. I think it might be bit more subtle. Its at least worth trying